config/can_config.h

```diff
@@ -46,8 +46,14 @@ typedef enum CANID {
   REPORT_CLAW_MOTION,
   FORCE_SENSOR_VALUE,
 
+  SET_GIMBAL_PAN_POS = 0x780,
```

Comment:
Note that these settings will need to be moved in #114 once this is merged @younesr1